WordPress/src/main/java/org/wordpress/android/ui/prefs/SmoothScrollLinearLayoutManager.java

```diff
@@ -22,7 +22,7 @@ public SmoothScrollLinearLayoutManager(Context context, int orientation, boolean
     public void smoothScrollToPosition(RecyclerView recyclerView, RecyclerView.State state, int position) {
         final View firstVisibleChild = recyclerView.getChildAt(0);
         final int itemHeight = firstVisibleChild.getHeight();
-        final int currentPosition = recyclerView.getChildPosition(firstVisibleChild);
+        final int currentPosition = recyclerView.getChildLayoutPosition(firstVisibleChild);
```

Comment:
I am not 100% sure about this, but I _think_ we want to use the `getChildAdapterPosition` here instead. Here is a quote from the docs for `getChildLayoutPosition`: 

> This position may not be equal to Item's adapter position if there are pending changes in the adapter which have not been reflected to the layout yet.

I don't think we should care about the pending changes since the scroll will start immediately on the main thread and we'd want that to be calculated with the current positions of the items. At the very least `getChildAdapterPosition` would preserve the current behavior since the deprecated `getChildPosition` is internally calling `getChildAdapterPosition` anyway.